cmake/win32/FixupBundle.gen.cmake

```diff
@@ -0,0 +1,27 @@
+include(BundleUtilities)
+
+set(WIN_SYSTEM_LIBS "^(KERNEL32|MSVCRT|ADVAPI32|OLE32|SETUPAPI|RPCRT4|WINMM|GDI32|USER32|MPR|COMCTL32|OPENGL32|UXTHEME|WS2_32|VERSION|SHELL32|DWMAPI|DSOUND|IMM32|OLEAUT32)\\.DLL")
+
+function(gp_resolved_file_type_override resolved_file type_arg)
+    string(TOUPPER ${resolved_file} resolved_file)
+    if (resolved_file MATCHES ${WIN_SYSTEM_LIBS})
+        set(${type_arg} "system" PARENT_SCOPE)
+    else()
+        set(${type_arg} "local" PARENT_SCOPE)
+    endif()
+endfunction()
+
+function(gp_resolve_item_override context item exepath dirs resolved_item_var resolved_var)
+    string(TOUPPER ${item} item)
+    if (item MATCHES ${WIN_SYSTEM_LIBS})
+        set(${resolved_var} 1 PARENT_SCOPE)
+    endif()
+endfunction()
+
+# Override tool for mingw.
+if (NOT "@MSVC@")
+    set(gp_tool "objdump")
+endif()
```

Comment:
I think `GetPrerequisites` already do this for MinGW.

Reply:
it uses `ldd` on linux - always. on windows with mingw it would use objdump.

Reply:
> it uses `ldd` on linux

When cross-compiling, it uses `objdump`.

Reply:
See for yourself; without those lines:
```
Run CPack packaging tool...
CPack: Create package using NSIS
CPack: Install projects
CPack: - Run preinstall target for: lmms
CPack: - Install project: lmms
CMake Error at /usr/share/cmake-3.10/Modules/GetPrerequisites.cmake:838 (message):
  /usr/bin/ldd failed: 1

Call Stack (most recent call first):
  /usr/share/cmake-3.10/Modules/BundleUtilities.cmake:628 (get_prerequisites)
  /usr/share/cmake-3.10/Modules/BundleUtilities.cmake:903 (get_bundle_keys)
  /lmms/build/FixupBundle.cmake:26 (fixup_bundle)
  /lmms/build/src/cmake_install.cmake:50 (include)
  /lmms/build/cmake_install.cmake:38 (include)


CPack Error: Error when generating package: lmms
Makefile:129: recipe for target 'package' failed
make: *** [package] Error 1
```
Also, from the cmake [docs](https://cmake.org/cmake/help/latest/module/GetPrerequisites.html):
```
It uses various tools to obtain the list of required shared library files:
dumpbin (Windows)
objdump (MinGW on Windows)
ldd (Linux/Unix)
otool (Mac OSX)
```

Reply:
Oh, it seems like `WIN32` is defined only in the build step, not in the install step.

Reply:
> Can we test that against AppVeyor as well?

Let's try. why not :D